app/src/main/res/navigation/nav_graph.xml

```diff
@@ -28,4 +28,9 @@
             android:name="movieId"
             app:argType="integer"/>
     </fragment>
+    <fragment
+        android:id="@+id/moreFragment"
+        android:name="com.antosik.benchproject.app.more.view.MoreFragment"
+        android:label="moreFragment"
```

Comment:
`label` should be set from string resources

Reply:
I changed it and in all other places in `nav_graph.xml` file